pkg/core/redis/context.go

```diff
@@ -0,0 +1,43 @@
+package redis
+
+import (
+	"context"
+
+	"github.com/gomodule/redigo/redis"
+)
+
+type contextKeyType struct{}
+
+var contextKey = contextKeyType{}
+
+type Context interface {
```

Comment:
Seems unused in this commit.

Reply:
It's some old prototyping code, will remove it.